backend/api/tests/asset_factory.py

```diff
@@ -324,19 +383,24 @@ def create_computetask(
             TaskDataSamples.objects.create(compute_task_id=key, data_sample_id=data_sample, order=order)
         compute_task.refresh_from_db()
 
-    for position, input in enumerate(compute_task.algo.inputs.all().order_by("identifier")):
-        ComputeTaskInput.objects.create(
-            task=compute_task,
-            identifier=input.identifier,
-            asset_key=INPUT_ASSET_KEY,
```

Comment:
This constant made the test data invalid. This issue was raised with the new `ComputeTaskInputAsset` which requires valid asset keys.